testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BasicSamlTest.java

```diff
@@ -156,7 +156,7 @@ public void testNoPortInDestination() throws Exception {
 
     @Test
     public void testExplicitPortInDestination() throws Exception {
-        testWithOverriddenPort(Integer.valueOf(System.getProperty("auth.server.http.port")), Response.Status.OK, containsString("login"));
+        testWithOverriddenPort(Integer.valueOf(System.getProperty("auth.server.https.port")), Response.Status.OK, containsString("login"));
```

Comment:
This needs to be conditioned on whether ssl tests are enabled or not.

Reply:
Done